sw/host/rom_ext_image_tools/signer/src/main.rs

```diff
@@ -122,6 +122,10 @@ fn update_image_manifest(
         ..Default::default()
     };
 
+    // Code region must be 32-bit aligned due to PMP constraints.
+    ensure!(image.manifest.code_start % 4 == 0, "Unaligned code start.");
```

Comment:
nit: do you mind defining a tiny function (can be nested) for the alignment check?

Reply:
Added the `is_word_aligned` function.